src/applications/ivc-champva/10-7959C/chapters/formSignature.js

```diff
@@ -27,3 +44,20 @@ export const formSignatureSchema = {
     },
   },
 };
+
+export const applicationEmailSchema = {
+  uiSchema: {
+    ...titleUI(
+      'Your email address',
+      `We'll use this email address to send you information about the form.`,
+    ),
+    applicantEmail: emailUI(),
+  },
+  schema: {
+    type: 'object',
+    required: ['applicantEmail'],
+    properties: {
+      applicantEmail: emailSchema,
+    },
+  },
+};
```

Comment:
With the inclusion of email we'll want to add the email property into the `primaryContactInfo` object [in the submit transformer](https://github.com/department-of-veterans-affairs/vets-website/blob/4cb5b2c08493835088a44e98684c021c0437eb07/src/applications/ivc-champva/10-7959C/config/submitTransformer.js#L11) as well.